### PR TITLE
Add new transit providers

### PR DESCRIPTION
**Pull request by randytorres**

### CODE REVIEW

**Comment by randytorres** [comment]
I still need to merge https://github.com/API-market/oreid-js/pull/15 and deploy a new version of eos-auth before this will work.

File: examples/react/src/App.js [hunk 1]
```diff
@@ -391,6 +405,26 @@ renderLoginButtons() {
...
+      />
+      <LoginButton provider='simpleos'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("simpleos")}
+      />
+      <LoginButton provider='keycat'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("keycat")}
```
Comment (traylewin) [inline examples/react/src/App.js]: I get "Cannot initiate a session, invalid wallet provider or wallet provider ID was passed" for keycat. Pretty sure I have 3.1.2 of eos-transit and the right transit providers installed. Do you get this same error?

File: examples/react/src/App.js [hunk 2]
```diff
@@ -391,6 +405,26 @@ renderLoginButtons() {
...
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("portis")}
+      />
+      <LoginButton provider='whalevault'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("whalevault")}
+      />
+      <LoginButton provider='simpleos'
```
Comment (traylewin) [inline examples/react/src/App.js]: I get "Failed to connect to simpleos wallet: TypeError: Failed to fetch" 
Don't think we should ever get this error - unless it's in the provider doing the fetch. Is that where it is trying to fetch?

File: examples/react/src/App.js [hunk 3]
```diff
@@ -391,6 +405,26 @@ renderLoginButtons() {
           logoStyle={{marginLeft:0}}
           onClick={()=>this.handleLogin("lynx")}
       />
+      <LoginButton provider='portis'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("portis")}
```
Comment (traylewin) [inline examples/react/src/App.js]: I get "Cannot initiate a session, invalid wallet provider or wallet provider ID was passed" for portis. Do you get this too?

File: examples/react/src/App.js (hunk 1 above)
Comment (randytorres) [reply examples/react/src/App.js]: Yes same error for me

File: examples/react/src/App.js (hunk 2 above)
Comment (randytorres) [reply examples/react/src/App.js]: Yes that error is inside of the provider. I've tried with http/https but it doesn't make a difference. It's worth noting that the simpleos website says this functionality is in beta.

File: examples/react/src/App.js (hunk 3 above)
Comment (randytorres) [reply examples/react/src/App.js]: You'll have to add a `DappId` when initializing the provider, you can get one from Portis.io. The portis modal pops up for me but after submitting my email/password it errors out.

**Comment by randytorres** [comment]
Whalevault is the only one that works correctly. The others all error out.

**Pull request merged**